model/constants.py

```diff
@@ -0,0 +1,260 @@
+# encoding: utf-8
+# DataSourceConstants, EditionConstants, IdentifierConstants, LinkRelations,
+# MediaTypes
```

Comment:
It's not clear why some constants were moved here and others weren't. Are these the most widely used outside their own classes?

Reply:
Yeah, these are the ones where they had to be in a separate module to avoid circular imports. I agree that we should move all the constants in here eventually.

Reply:
I filed https://jira.nypl.org/browse/SIMPLY-1252 as a follow-up ticket.